File: population.cs
using System;
# Kata URL : https://www.codewars.com/kata/563b662a59afc2b5120000c6
class Arge {

    public static int NbYear(int p0, double percent, int aug, int p) {

      int year = 0;
      double more = p0;
      double pourcent = percent / 100;
      do
        {
          more = (int)more + more * pourcent + aug;
          year +=1;
        } while (p > more);
      return year;
    }
}

File: population.py
def nb_year(p0, percent, aug, final):
    """Kata URL : https://www.codewars.com/kata/563b662a59afc2b5120000c6"""
    year = 0
    more = p0
    pourcent = percent / 100
    while more < final:
        more = int(more + more * pourcent + aug)
        year += 1
    return year
